### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by hg-ghggggg** [T-vK/alexa-jammer#1]
Hi!
Thank you very much for your explanation, it's quite clear.
I read your comments in the "wearable jammer bracelet" and I would like to know if your prototype is still ok ? I understand that when you try with a 25Khz transducer and the cheap Mini power amplifier board, the prototype was not very efficient.
Also, I have an Arduino Uno, I wonder if I can use it instead of the arduino pro micro used in the tutorial.

Thanks again!!
I would like to make my own alexa-jammer :)

Bye

**Comment by T-vK** [comment]
My prototype is okay, but range is somewhat limited. I could of course use an amplifier with more power, but I'm a bit worried about hearing loss because I can't measure how much decibels the device is outputting.

You can totally replace the Arduino Pro Micro with an Arduino Uno or any other Arduino for that matter.

**Comment by hg-ghggggg** [comment]
Ok, thanks
I will to try to do that. First, I will try without amplifier (like you did in the picture), because the shipping of the amplifier is very long.

![image](https://user-images.githubusercontent.com/75981456/102083854-82d1fd00-3e14-11eb-99c9-55fbb4ee4fa6.png)

Thanks !

**Comment by hg-ghggggg** [comment]
Hello, I have received all my parts and after installing my amplifier I notice that the sound intensity is lower than when I connect to a 1Kohm resistor. I bought a PAM 8403 3W amplifier (without potentiometer).
Do you know why ?

Thanks.

**Comment by T-vK** [comment]
No, I don't know. You didn't even tell me what the resistor is connected to.
